apps/aggiemap/src/app/modules/directives/accordion.directive.ts

```diff
@@ -67,7 +68,8 @@ export class AccordionDirective implements OnChanges, OnInit, OnDestroy {
     if (this.resize) {
       this.mutationObserver = new MutationObserver((mutations: MutationRecord[]) => {
         this.setAccordionHeight();
-      }).observe(this.el.nativeElement, {
+      });
+      this.mutationObserver.observe(this.el.nativeElement, {
```

Comment:
Functional change,

`.observe(...)` returns void. So I think this is what this block of code was supposed to look like